### PR TITLE
Heavy changes

### PR DESCRIPTION
**Pull request by CitTheDev**
I put all the main code into its separate `src` folder so the root folder looks more clean. I also added type definitions for typescript users as the previous approach did not work as intended. I also removed the need of the `fs` package as the code directly imports the json file via `require`. All variables that were created with the `var` keyword have also been replaced to use `let` or `const`. I also removed the `alignLeft`, `alignRight`, and `alignCenter` methods as they are internally using the `align` method to set the alignment, so having these was useless.

### CODE REVIEW

**Comment by cit999** [comment]
@AllMightySauron When you have time please have a look at this.
Thanks :)

**Comment by AllMightySauron** [comment]
Sure, I'll be back on Monday. Just give me a few days to review it...

File: package.json [hunk 1]
```diff
@@ -19,21 +19,10 @@
       "github": "https://github.com/AllMightySauron"
     }
   ],
-  "main": "index.js",
+  "main": "src/index.js",
   "engines": {
     "node": ">=11.14.0"
   },
-  "scripts": {
```
Comment (AllMightySauron) [inline package.json:26]: Why did you remove de QA dev dependencies and scripts for **unit testing** and **coverage**?

File: package.json [hunk 2]
```diff
@@ -19,21 +19,10 @@
       "github": "https://github.com/AllMightySauron"
     }
   ],
-  "main": "index.js",
+  "main": "src/index.js",
```
Comment (AllMightySauron) [inline package.json:22]: No problems there...
